code/index.Rmd

```diff
@@ -39,35 +41,40 @@ Reallocating roadspace is just one of the measures available for transport autho
 
 We have used several types of data. The cycling potential of each road is estimated according the Propensity to Cycle Tool's Government Target scenario, which represents the government target to double cycling levels by 2025. Our figures include both travel to work and travel to school, and are based on 2011 Census data. Data on road widths and number of lanes is derived from the Cycling Infrastructure Prioritisation Toolkit.
 
-### Map layers
+# Map layers
 
-The maps show five layers. 
+The maps contain five layers of data that overlay the basemaps. 
 
-1) Key network:
+## 1. Key network:
 
-- we have created this to represent the most important cycle corridors. For this layer, we do not take into account the width of the roads, but simply represent them as continuous corridors. This underscores the importance of ensuring that cycle routes are continuous and do not 'give up' at awkward junctions. To derive this layer, we have selected all road segments with cycle potential in the 85th percentile and above, which also belong to classified trunk roads, A roads or B roads. These segments are then joined together to form a continuous network.
+This layer is designed to represent a cohesive network of direct routes.
+It shows what a joined-up cycle network could look like if there were no constraints on road space, along some of the most important corridors.
```

Comment:
Heads-up @joeytalbot I advocate line breaks per sentence in markdown files. Makes for easier tracking of changes. We used it in our book Geocomputation with R after finding 'per paragraph' breaks too long.